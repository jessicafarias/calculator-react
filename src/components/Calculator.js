const Calculator = () => (
  <div>
    <h1>calculator</h1>
  </div>
);

export default Calculator;
