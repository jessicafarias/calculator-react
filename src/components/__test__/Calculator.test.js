import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import '@testing-library/jest-dom';
import Calculator from '../Calculator';
import TestRenderer from 'react-test-renderer'; // ES6

describe('Display Calculator page', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(<BrowserRouter><Calculator /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});