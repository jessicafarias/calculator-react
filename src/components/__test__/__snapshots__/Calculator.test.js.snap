// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Calculator page renders correctly 1`] = `
<div
  className="bg-image3"
>
  <div
    className="bg-black"
  >
    <h1
      className="calculator-text "
    >
       CALCULATOR 
    </h1>
    <p>
      Letâ€™s do some math!
    </p>
  </div>
  <div
    className="calculator-base"
  >
    <div
      className="text-gray"
    >
      <h1>
        0
      </h1>
    </div>
    <div
      className="calculator-base"
    >
      <div
        className="resize"
      >
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          AC
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          +/-
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          %
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#F5913E",
              "width": "25%",
            }
          }
          type="button"
        >
          /
        </button>
      </div>
      <div
        className="resize"
      >
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          7
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          8
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          9
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#F5913E",
              "width": "25%",
            }
          }
          type="button"
        >
          X
        </button>
      </div>
      <div
        className="resize"
      >
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          4
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          5
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          6
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#F5913E",
              "width": "25%",
            }
          }
          type="button"
        >
          -
        </button>
      </div>
      <div
        className="resize"
      >
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          1
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          2
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          3
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#F5913E",
              "width": "25%",
            }
          }
          type="button"
        >
          +
        </button>
      </div>
      <div
        className="resize"
      >
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "50%",
            }
          }
          type="button"
        >
          0
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#E0E0E0",
              "width": "25%",
            }
          }
          type="button"
        >
          .
        </button>
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#F5913E",
              "width": "25%",
            }
          }
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
